docs/source/learn/core_notebooks/pymc_aesara.ipynb

```diff
@@ -0,0 +1,1883 @@
+{
```

Comment:
<p>new_w.name shoud be log of exp and not exp of log!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Reply:
good catch! thanks!